Makefile

```diff
@@ -1,10 +1,12 @@
 build-wasm-plugin:
+	cd modsecurity-wasm-filter/wasmplugin; sed -i 's/envoy-wasm-modsecurity-dynamic/envoy-wasm-modsecurity/' Dockerfile; sed -i 's/envoy-wasm-modsecurity-dynamic/envoy-wasm-modsecurity/' Makefile
```

Comment:
Tried to automate the build of the static version reverting afterwards everything to it's original state